x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.js

```diff
@@ -172,8 +172,8 @@ export function convertESShapeToGeojsonGeometry(value) {
       geoJson.type = GEO_JSON_TYPE.MULTI_POLYGON;
       break;
     case 'geometrycollection':
-      geoJson.type = GEO_JSON_TYPE.GEOMETRY_COLLECTION;
-      break;
+    case GEO_JSON_TYPE.GEOMETRY_COLLECTION:
+      throw new Error('Should not pass geometrycollection to convertESShapeToGeojsonGeometry');
```

Comment:
Does this need to be i18n message, circle throws error in same switch and its message is i18n